### DISCUSSION TITLE
从0.4.2 版本开始， inter Mac 版本没有了

### DISCUSSION THREAD

**Discussion by leohooho** [nextai-translator/nextai-translator#1490] [Q&A]
OpenAI.Translator_0.4.1_x64.dmg 最近的两个版本 没有看到了

**Comment by WhiteDG** [comment]
从 0.4.2 开始本来应该构建 `x86_64` 的任务 [build-tauri (macos-latest, x86_64, x86_64-apple-darwin)](https://github.com/openai-translator/openai-translator/actions/runs/8810018518/job/24181764567#logs) 构建成了 `aarch64` 版本，看日志应该是 `install Rust nightly` 这个步骤更新了，默认使用 aarch64 版本了。

0.4.2 `Build Tauri App` 日志 👇
![image](https://github.com/openai-translator/openai-translator/assets/9566166/09a6caec-4db3-43e2-9895-15b6b7e40e74)

0.4.2 `install Rust nightly` 日志 👇
![image](https://github.com/openai-translator/openai-translator/assets/9566166/b7f01ace-1c9c-45f7-8d83-9fc69badd707)

0.4.1 `install Rust nightly` 日志 👇
![image](https://github.com/openai-translator/openai-translator/assets/9566166/e8e1ce8f-5062-4797-8d43-c3d9b7f2fda9)

**Comment by yetone** [comment]
最新版已有 x86_64 的安装包
